web/src/routes/(user)/map/+page.svelte

```diff
@@ -8,17 +8,25 @@
 		isViewingAssetStoreState,
 		viewingAssetStoreState
 	} from '$lib/stores/asset-interaction.store';
+	import { api, MapMarkerResponseDtoDecorator } from '@api';
+	import { onMount } from 'svelte';
+	import { browser } from '$app/environment';
 
 	export let data: PageData;
+	let mapMarkers: MapMarkerResponseDtoDecorator[] = [];
 
 	let initialMapCenter: [number, number] = [48, 11];
 
-	$: {
-		if (data.mapMarkers.length) {
-			let firstMarker = data.mapMarkers[0];
-			initialMapCenter = [firstMarker.lat, firstMarker.lon];
+	onMount(async () => {
+		mapMarkers = MapMarkerResponseDtoDecorator.map(data.mapMarkers);
```

Comment:
Map is a JavaScript standard array function and calling this custom method the same thing is a bit misleading/confusing. Maybe we should use an actual array.map here. So like response.map(item => Custom class.from(item)) and call the method that converts from plain to instance `from` instead of `map`. Or, just do new CustomClass(item).